src/actions/PatientActions.js

```diff
@@ -18,7 +18,7 @@ export const PATIENT_ACTIONS = {
 };
 
 const closeModal = () => ({ type: PATIENT_ACTIONS.COMPLETE });
-const createPatient = () => ({ type: PATIENT_ACTIONS.PATIENT_CREATION });
+const createPatient = patient => ({ type: PATIENT_ACTIONS.PATIENT_CREATION, payload: { patient } });
```

Comment:
This is for creating a patient in normal dispensing (because I guess I thought it was a good idea to have name actions and patient actions 🤔 )

This is setting a 'default name' as the one being edited now rather than sorting it out when saving, so that there is a stable ID to use for the name note.